source/api_documentation/transactions_api/intro.rst

```diff
@@ -51,5 +51,3 @@ Please note, the "to" and "from" date range parameters are both necessary, provi
 Example:
 
 Send request 1:
-
-
```

Comment:
Were these lines needed for spacing? Just making sure your IDE didn't remove these automatically/unintentionally.